libazureinit/Cargo.toml

```diff
@@ -16,7 +16,7 @@ tokio = { version = "1", features = ["full"] }
 serde-xml-rs = "0.6.0"
 xml-rs = "0.8.13"
 serde_json = "1.0.96"
-nix = "0.26.2"
+nix = {version = "0.28.0", features = ["user"]}
```

Comment:
if we add features "fs" (in addition to "user") we don't have to make change to the user.rs file

Reply:
True, thanks. Fixed.